### PR TITLE
[analyze] Add zjs_common.json that always gets pulled in

### LINKED ISSUE #1260: Recent updates broke build for most Zephyr boards
Previously, (minimal) Zephyr.js could be built for any Zephyr board. Recently, build for any board besides handful explicitly "supported" is broken:

~~~
In file included from /mnt/hdd/projects-3rdparty/Embedded/Zephyr/zephyr.js/src/zjs_modules.c:21:0:
/mnt/hdd/projects-3rdparty/Embedded/Zephyr/zephyr.js/src/zjs_modules.c: In function ‘native_print_handler’:
/mnt/hdd/projects-3rdparty/Embedded/Zephyr/zephyr.js/src/zjs_util.h:39:23: error: implicit declaration of function ‘free’ [-Werror=implicit-function-declaration]
 #define zjs_free(ptr) free(ptr)
                       ^
/mnt/hdd/projects-3rdparty/Embedded/Zephyr/zephyr.js/src/zjs_modules.c:133:5: note: in expansion of macro ‘zjs_free’
     zjs_free(str);
     ^~~~~~~~
~~~

That's because Zephyr.js requires CONFIG_NEWLIB_LIBC, but instead of being defined globally for the application, it's now moved into individual zjs_qemu_x86.json, zjs_qemu_x86.json, etc.

I'm looking how to re-enable it globally.

### PR DESCRIPTION
**Pull request by grgustaf**
This includes base configuration common to all platforms.

Fixes #1260.

Signed-off-by: Geoff Gustafson <geoff@linux.intel.com>

### CODE REVIEW

**Comment by pfalcon** [comment]
I tested a few boards and this indeed fixes builds for them, thanks!

**Comment by jprestwo** [comment]
+1 LGTM

File: src/zjs_common.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "module": "common",
+    "zephyr_conf": {
+        "all": [
+            "CONFIG_STDOUT_CONSOLE=y",
+            "CONFIG_NEWLIB_LIBC=y",
+            "CONFIG_FLOAT=y",
+            "CONFIG_NANO_TIMERS=y",
```
Comment (jimmy-huang) [inline src/zjs_common.json]: I don't these these NANO_ flags are used anymore, maybe you should find the new equivalent if they exists

**Comment by jimmy-huang** [comment]
+1

**Pull request merged**